Manager/PasteboardManager.m

```diff
@@ -183,11 +183,12 @@ - (void)updatePasteboardWithItem:(PasteboardItem *)item fromHistoryWithKey:(NSSt
     }
 
     // the given item gets removed, as the pasteboard is updated with the same item
+    [_pasteboard changeCount];
```

Comment:
Bugfix: force the pasteboard to update itself.

Reply:
This already happens, no? After selecting an item from the history view, the pasteboard's content changes and thus a count change occurs.

Reply:
Yes. But I found this line actually resolves a bug that the pasteboard will not get synconized sometimes.